Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -1142,7 +1142,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 #if ENABLE(IOS_TOUCH_EVENTS)
     void resetPotentialTapSecurityOrigin();
     void handlePreventableTouchEvent(NativeWebTouchEvent&);
-    void handleUnpreventableTouchEvent(const NativeWebTouchEvent&);
+    void handleUnpreventableTouchEvent(NativeWebTouchEvent&);
```

Comment:
Can we add const to updateTouchEventTracking instead of removing const here?

Reply:
I don't think so? I needed to remove const here because we are passing the event to `sendUnpreventableTouchEvent` which may modify the event in the lambda. I guess I could create a mutable copy of the event and so that we could keep the const. I'll do that.

Reply:
> I don't think so? I needed to remove const here because we are passing the event to sendUnpreventableTouchEvent which may modify the event in the lambda.

Note that the event in the lambda is a copy of the event you passed to the function. Therefore that inner event is not const if you mark the lambda as mutable.

Reply:
> Note that the event in the lambda is a copy of the event you passed to the function, because you captured it by value. Therefore that inner event is not const as long as the lambda is marked as mutable.

I don't think that's right. `event` doesn't lose const after being passed into the lambda capture by value, I get a compile error when I try. If I pass it as `[event = event]` it does work.